src/modules/base/header/header.js

```diff
@@ -153,7 +153,7 @@
 
                 for (i = 0; i < len; i++) {
                     if (id === data[i].clusterId) {
-                        return data[i].clusterId;
+                        return data[i].name;
```

Comment:
what if name is not present?

Reply:
@cloudbehl, this condition is handled in cluster-store `temp.name = (cluster.short_name && cluster.short_name !== "None") ? cluster.short_name : cluster.integration_id;`. If short_name wont be present, integration_id will be assigned to name.

Reply:
ok  good